pkg/kubelet/cm/cpumanager/cpu_manager_test.go

```diff
@@ -1118,7 +1129,7 @@ func TestCPUManagerHandlePolicyOptions(t *testing.T) {
 			}
 			defer os.RemoveAll(sDir)
 
-			_, err = NewManager(testCase.cpuPolicyName, testCase.cpuPolicyOptions, 5*time.Second, machineInfo, cpuset.NewCPUSet(), nodeAllocatableReservation, sDir, topologymanager.NewFakeManager())
+			_, err = NewManager(testCase.cpuPolicyName, testCase.cpuPolicyOptions, 5*time.Second, machineInfo, cpuset.NewCPUSet(), nodeAllocatableReservation, sDir, topologymanager.NewFakeManager(), false)
```

Comment:
Would it be possible to set a flag to false above and pass that in, instead of "false", for ease of reading?  I see above you have testcase.strictCPUObserved set to true, so should be similar here?